pkg/infrastructure/services/resources.go

```diff
@@ -185,7 +212,14 @@ func (s *serviceDeployer) getDeployedResources(instance v1alpha1.KogitoService)
 	if infrastructure.IsStrimziAvailable(s.client) {
 		objectTypes = append(objectTypes, &kafkabetav1.KafkaTopicList{})
 	}
-	return reader.ListAll(objectTypes...)
+
+	resources, err = reader.ListAll(objectTypes...)
+	if err != nil {
+		return
+	}
+
+	ExcludeAppPropConfigMapFromResource(instance.GetName(), resources)
```

Comment:
If `configMap` is not in the  `objectTypes` list we still need this function? I understand that if we have other kind of configMaps that we control we will need to remove only the properties one, which makes sense. But in this case that we don't have the `operator-utils` will fetch it either way because of the owner reference?

Reply:
The `configmaps` won't be fetched because it is not in the `objectTypes` list. Added this code here, just in case we will introduce other `configmaps` in future. With or without this line of code makes no difference for now. I can remove it if you think we should take care of it in future.

Reply:
Let's leave it there since we won't have a configMap in the list, thus the function will return before entering in the loop. Won't be a problem IMO.